logisland-components/logisland-processors/logisland-processor-computer-vision/src/main/java/com/hurence/logisland/cv/utils/ClojureWrapper.java

```diff
@@ -0,0 +1,144 @@
+package com.hurence.logisland.cv.utils;
+
+import clojure.lang.RT;
+import clojure.lang.Var;
+import org.apache.commons.io.IOUtils;
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.scijava.nativelib.NativeLoader;
+
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.*;
+
+import static com.hurence.logisland.cv.utils.CVUtils.toBI;
+import static com.hurence.logisland.cv.utils.CVUtils.toMat;
+
+
+public class ClojureWrapper {
```

Comment:
What is the use of this class ? If this is just a local job for your test with Clojure this should not be commited or should be moved in test so this is not packaged in logisland.
The resources associated (images) should be moved into test resources as well.